building/build-debs/homeworld-admin-tools/src/verify.py

```diff
@@ -75,7 +75,9 @@ def check_keygateway():
     try:
         access.call_keyreq("check")
     except subprocess.CalledProcessError as e:
-        command.fail("keygateway check failed: %s" % e)
+        command.fail("keygateway check failed: %s" % e,
+            '** do you have valid kerberos tickets? **' if
+            e.returncode == 1 else None)
```

Comment:
I would avoid using an inline `if` here, for the sake of readability, but it doesn't matter that much.